tests/tool_use/test_tool_calls.py

```diff
@@ -119,7 +119,7 @@ async def test_tool_call_and_choice(client: openai.AsyncOpenAI):
     assert isinstance(streamed_args.get("city"), str)
     assert isinstance(streamed_args.get("state"), str)
     assert streamed_args.get("city") == "Dallas"
-    assert streamed_args.get("state") == "TX"
+    assert parsed_arguments.get("state") == "TX"
```

Comment:
Shouldn't this be `streamed_arguments` instead of `parsed_arguments`?

Reply:
oh yes, i will fix it, thanks.

Reply:
I have fixed this in the commit 647db0d